draft-irtf-cfrg-rsa-blind-signatures.md

```diff
@@ -740,75 +632,240 @@ This document makes no IANA requests.
 
 # Test Vectors
 
-This section includes test vectors for the blind signature protocol defined in {{internal}}.
-It does not include test vectors based on the external interface in {{salted-interface}}.
+This section includes test vectors for the blind signature protocol defined in {{core-protocol}}.
 The following parameters are specified for each test vector:
 
 - p, q, n, e, d: RSA private and public key parameters, each encoded as a hexadecimal string.
-- msg: Messsage being signed, encoded as a hexadecimal string. The hash is computed using SHA-384.
+- msg: Input messsage being signed, encoded as a hexadecimal string. The hash is computed using SHA-384.
+- msg\_salt: Message randomizer salt, encoded as a hexadecimal string. This is only present for variants
+  that use message randomization.
 - salt: Randomly-generated salt used when computing the signature. The length (sLenInBytes) is either 48 or 0 bytes.
 - inv: The message blinding inverse, encoded as a hexadecimal string.
+- signed\_msg: The message actually signed. If the variant does not use message randomization, this is equal
```

Comment:
I couldn't find this variable in the description of the algorithms of this document.

Reply:
I'll use random_msg to match the message randomization section.